projects/step-frontend/src/lib/modules/parameter/parameters-list/parameters-list.component.scss

```diff
@@ -2,6 +2,12 @@
 
 :host {
   @include core.table-container;
+
+  step-table {
+    ::ng-deep td.cdk-column-value {
```

Comment:
from what I know ::ng-deep should not be used as it is deprecated, the fact that it still works is not something we should be relying on in my opinion, the proper way to handle this would be to use ViewEncapsulation.None and then instead of :host you would use the component selector e.g. step-parameters-list { ... }

Reply:
I'm not sure, ViewEncapsulation.None is quite dangerous. In my opinion ng-deep inside of :host is the better approach. I guess they only deprecate it so people double check it is really encapsulated in :host.